### PR TITLE
Minor updates to business model page, include self-serve description 

### PR DESCRIPTION
**Pull request by it33**
WIP: Updates to business model context

### CODE REVIEW

File: company/about-mattermost/business-model.md [hunk 1]
```diff
@@ -64,7 +64,11 @@ A subscription license to [Mattermost Professional](https://mattermost.com/prici
 
 As the deployment of Mattermost grows within an organization, typically past 250 or 500 users, the customer's procurement organization may be interested in advanced features and customization available in our [Mattermost Enterprise](https://mattermost.com/pricing/) offering.
 
-# Sales motions
+# Self-served motions
+
+For Mattermost Professional, we have self-served options to purchasing subscription licenses by credit card. For self-serve purchases, cloud-hosted Mattermost Professional is purchased with a monthly charge per paid user, and the self-hosted version is purchased with an annual contract.
```
Comment (justinegeffen) [inline company/about-mattermost/business-model.md]:
```suggestion
For Mattermost Professional, we have self-serve options to purchasing subscription licenses by credit card. For self-serve purchases, cloud-hosted Mattermost Professional is purchased with a monthly charge per paid user, and the self-hosted version is purchased with an annual contract. 
```

**Comment by justinegeffen** [comment]
@chenilim, gentle reminder on this. Thank you! :)

**Review by chenilim — DISMISSED** [review]
Sorry for the delay. LGTM.

**Comment by cwarnermm** [comment]
@chenilim - Any concerns about publishing this PR?

**Comment by chenilim** [comment]
Sorry for the delay @cwarnermm. From a quick review, the notes about self-serve are no longer accurate (e.g. monthly subscriptions have been replaced with annual).

**Comment by chenilim** [comment]
I propose removing the last sentence: "For self-serve purchases, cloud-hosted Mattermost Professional is purchased with a monthly charge per paid user, and the self-hosted version is purchased with an annual contract.". The rest LGTM.

**Pull request merged**